lib/LambdaHelper.js

```diff
@@ -16,6 +25,42 @@ var _getFunctionHash  = function (func) {
     // TODO : add config in hash
 };
 
+var _getModuleHash = function (modulePath, handlerName, deps, configs) {
+    var configsAsStr = JSON.stringify(configs),
+        identifier = [MODULE_HASH_PREFIX, modulePath, handlerName, deps, configsAsStr].join('_');
+
+    return md5(identifier); // TODO : add config in hash?
+};
+
+var _loadModule = function(moduleName) {
+    return require(moduleName); // TODO throw better errors if require fails
+};
+
+var _getDependencyFolder = function(dep) {
+    var depFolder = path.dirname(require.resolve(dep)) + path.sep;
+    var subFolders = depFolder.split(path.sep);
+    var folders = subFolders.splice(0, subFolders.indexOf('node_modules') + 2);
+    return folders.join(path.sep); // TODO throw better errors if require fails
+};
+
+var _addFolderRecursiveToZipNode = function(folder, zipNode) {
+    var basename = path.basename(folder);
+    var node = zipNode.folder(basename);
+    var files = fs.readdirSync(folder);
+
+    files.forEach(function(file) {
+        var fullPath = path.join(folder, file);
+        if(fs.statSync(fullPath).isDirectory())
+            _addFolderRecursiveToZipNode(fullPath, node);
+        else
+            node.file(file, fs.readFileSync(fullPath));
```

Comment:
I'll add a TODO. This is tricky because you never know if the lib uses the .txt file or not